_docs/configuration_files/application_json.rst

```diff
@@ -68,6 +68,13 @@ The amount of memory to give a Lambda function
     | *Default*: ``"128"``
     | *Units*: Megabytes
 
+``lambda_role``
+***************
+
+Override the default generated IAM Role name.
+
+    | *Default*: ``"${stack}_${app}_role"``
```

Comment:
I would suggest to add the word 'lambda' to the role name.
So we can differentiate between apps roles and lambda roles. That makes troubleshooting easier.